tests/rest_server_test.cc

```diff
@@ -110,10 +110,24 @@ TEST(rest_server_test, basic_test)
     {
         ASSERT_EQ(res->body, "ip6-localhost"); // count the passing test.
     }
-    else
+    else if (res->body == "localhost")
     {
         ASSERT_EQ(res->body, "localhost");
     }
+    else
+    {
+        const unsigned int my_max_hostname_len = 1024;
```

Comment:
Is there a reason why you didn't use `HOST_NAME_MAX` instead?

Reply:
I recall HOST_NAME_MAX from some past project (don't know where, don't know when etc.) is not always followed by the OS. Choosing a bigger number...